testnet/stacks-signer/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "stacks-signer"
+version = "0.0.1"
+license = "GPLv3"
+homepage = "https://github.com/blockstack/stacks-blockchain"
+repository = "https://github.com/blockstack/stacks-blockchain"
+edition = "2021"
+
+[[bin]]
+name = "stacks-signer"
+path = "src/main.rs"
+
+[lib]
+path = "src/lib.rs"    # The source file of the target.
+crate-type = ["lib"]   # The crate types to generate.
+
+[dependencies]
+sha3 = "0.10.6"
+serde = { version = "1.0", features = ["serde_derive"] }
+toml = "0.5.9"
+hashbrown = { version = "0.13", features = ["serde"] }
+frost = { git = "https://github.com/trust-machines/frost", branch = "multiple-parties-per-signer" }
+rand_core = { version  = "0.5"}
+tracing = "0.1.37"
```

Comment:
Have you confirmed that these dependencies are free of gratuitous `unsafe` code?

* `tracing`
* `tracing-subscriber`
* `ureq`

Reply:
ureq is clean, from the ureq Readme: "Ureq is in pure Rust for safety and ease of understanding. It avoids using `unsafe` directly. It [uses blocking I/O][blocking] instead of async I/O, because that keeps the API simple and keeps dependencies to a minimum. For TLS, ureq uses [rustls or native-tls](#https--tls--ssl)."

tracing I added just to have something for logging. grepping the tracing codebase finds lots of unsafe references. I'll take tracing out of Cargo.toml.